src/main/kotlin/com/dmdirc/App.kt

```diff
@@ -35,6 +35,7 @@ class MainApp : Application() {
             minHeight = 600.0
             scene = Scene(kodein.direct.instance<MainView>())
             show()
+            kodein.direct.instance<FocusManager>().start()
```

Comment:
Might be nicer to declare it an eagerSingleton and just stick a `.apply { start() }` in the kodein code? Unless it has to be done after show?

Reply:
Needs to be done after show it seems.